draft-ietf-acme-acme.md

```diff
@@ -2709,7 +2603,6 @@ Initial Contents
 | Label      | Identifier Type | ACME | Reference |
 |:-----------|:----------------|:-----|:----------|
 | http-01    | dns             | Y    | RFC XXXX  |
-| tls-sni-02 | dns             | Y    | RFC XXXX  |
```

Comment:
I agree with @kelunik that it would be good to have a tombstone here.  The standard practice is to have a "reserved" entry in the registry, so that you don't conflict later, but you also don't define it.  Here, I think the right row would be:

```
| tls-sni-02 | RESERVED | N | N/A |
```

Reply:
@bifurcation Shouldn't there be one for `tls-sni-01` by the same logic? Should I add both in this PR?

Reply:
I added both in 0bb71a0 - I can drop one if there's a reason we should only have a tombstone for tls-sni-02.

Reply:
Confirmed OOB this addresses @bifurcation's comment.